src/modules/Channel/components/SuggestedReplies/index.tsx

```diff
@@ -0,0 +1,43 @@
+import './index.scss';
+import React, { useState } from 'react';
+
+interface Props {
+  replyOptions: string[];
+  onSendMessage: ({ message }: { message: string }) => void;
+}
+
+const SuggestedReplies = ({ replyOptions, onSendMessage }: Props) => {
+  const [replied, setReplied] = useState<boolean>(false);
+
+  const onClickReply = (
+    event: React.MouseEvent<HTMLDivElement>,
+    option: string,
+  ) => {
+    event.preventDefault();
+    onSendMessage({ message: option });
+    setReplied(true);
+  };
+
+  if (replied) {
+    return null;
+  }
+
+  return (
+    <div className="sendbird-suggested-replies">
+      {replyOptions.map((option: string) => {
+        return (
+          <div
+            className="sendbird-suggested-replies__option"
+            id={option}
+            key={option}
```

Comment:
If the value can be set by the customer on the dashboard, it may be a good idea to add an index because we can't be sure it's unique. `key={index+option}`

Reply:
Sure it's been addressed too https://github.com/sendbird/sendbird-uikit-react/pull/825/commits/13206b3337445d1f7c1673aac7be6542674ea808